EpiAware/src/models.jl

```diff
@@ -1,98 +1,28 @@
 @model function log_infections(
     y_t,
-    ::Type{T} = Float64;
     epimodel::EpiModel,
-    latent_process,
+    latent_process;
+    latent_process_priors,
     transform_function = exp,
+    n_generate_ahead = 0,
     pos_shift = 1e-6,
-    α = missing,
-) where {T}
-
-    I_t = Vector{T}(undef, gen_length)
-    mean_case_preds = Vector{T}(undef, gen_length)
-    data_length = length(y_t)
-
-    α ~ Gamma(3, 0.05 / 3)
+    neg_bin_cluster_factor = missing,
+    neg_bin_cluster_factor_prior = Gamma(3, 0.05 / 3),
+)
+    #Prior
+    neg_bin_cluster_factor ~ neg_bin_cluster_factor_prior
```

Comment:
Does this mean we're sampling from a hyperdistribution when generating deviates? Or is there a single realization?